Documentation/compatibility/uri-unicode-scheme-only-relative.md

```diff
@@ -0,0 +1,40 @@
+## Support special relative URI notation when Unicode is present
+
+### Scope
+Edge
+
+### Version Introduced
+4.7.3
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+
+<xref:System.Uri> will no longer throw a NullReferenceException when calling <xref:System.Uri.TryCreate> on certain relative URIs containing Unicode.
+
+The simplest reproduction of the <xref:System.NullReferenceException> is below, with the two statements being equivalent:
+ ```C#
+bool success = Uri.TryCreate("http:%C3%A8", UriKind.RelativeOrAbsolute, out Uri href);
+bool success = Uri.TryCreate("http:è", UriKind.RelativeOrAbsolute, out Uri href);
+ ```
+To reproduce the <xref:System.NullReferenceException> the two following items must be true:
+- The URI must be specified as relative by prepending it with ‘http:’ and not followed by ‘//’.
+- The URI must contain percent-encoded Unicode or unreserved symbols.
+
+- [ ] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+Users depending on this behavior to disallow relative URIs should instead specify <xref:System.UriKind.Absolute> when creating a URI.
+
+### Affected APIs
+* `T:System.Uri.TryCreate`
```

Comment:
This is an overloaded method, so I think it needs to be \`Overload:System.Uri.TryCreate\`

Reply:
All of these should be addressed in the latest commit. Thanks for the suggestions!